### DISCUSSION TITLE
Harmonisierter Online-Zugang zu Grundstückinformation/Grundstückbesitzer:in

### DISCUSSION THREAD

**Discussion by sfkeller** [GeoUnconference/discussions#36] [Ideen / Idées]
Es soll ein zentraler harmonisierter Online-Zugang geben, um den Namen des Grundstückbesitzer:in in der Schweiz zu erfahren - und/oder aber ihn einfach kontaktieren zu können. Mit "zentral" meine ich nicht, dass alles über ein zentrales Monopol-System laufen muss; dafür gibt es ja Computer-Schnittstellen.

Das geht heute in einigen Kantonen, wie BS, BL, GL, SH, SO, UR, ZG, GE online (Anm. wäre schön wenn die Arbeitsgruppe unten eine Übersicht machen würde). Oder man muss mühsam eine Kontaktanfrage ans Grundbuchamt/Notariat machen. 

Das Grundbuch ist in der Kompetenz der Kantone; der Bund hat das Recht Empfehlungen abzugeben. Das GBV ist seit 2011 in Kraft. Es gibt seit 2021 eine Arbeitsgruppe "Grundstückinformation" unter der Leitung der swisstopo/GBA (vgl. www.cadastre.ch ). Diese Arbeitsgruppe entwirft Weisungen und Empfehlungen, die 2023 in Konsultation gehen sollen. 

Das geht alles recht langsam...: Gibt es Möglichkeiten dass zu Beschleunigen, beispielsweise mit Ziel Ende 2023?

**Comment by rastrau** [comment]
Danke für Deine Überlegungen und Fragen, Stefan. Die Frist für GeoUnconference 4-Themen [war der 2. August](https://twitter.com/GeoUnconference/status/1544191172159148032). Wir können das Thema also nicht mehr auf die Agenda heben, aber natürlich kann hier der Platz dennoch gerne für eine Diskussion genutzt werden 🚀
